### PR TITLE
C4 Suicide phrases now encompass most antagonists.

### PR DESCRIPTION
**Pull request by unit0016**
## About The Pull Request
Fun fact. Did you know the C4 has special suicideverb phrases for most antagonists?  That is, *most* antagonists. This adds a bunch in because someone noticed Heretic didn't have one special, and I noted quite a few minor antagonists didn't either.

## Why It's Good For The Game
The sheer absurdity of Santa Claus charging you with C4, shouting "FOR CHRISTMAS!" and exploding is worth it alone.
forchristmas
image

## Changelog
:cl:
expansion: Heretics (and several minor antagonists) now shout out their own batch of phrases when using the suicide verb with C4/X4.
/:cl:

### CODE REVIEW

**Comment by DuffCreeper** [comment]
Thanks Santa

**Comment by ExcessiveUseOfCobblestone** [comment]
Mobile posting warning

You need to dismantle the switch and put it on the datums (it can just pick one if you have multiple I don't care how you approach that so long as it's sane).

File: code/game/objects/items/grenades/plastic.dm [hunk 1]
```diff
@@ -141,6 +141,26 @@
...
+			message_say = "FOR THE CONSORTIUM!"
+		else if(our_guy.has_antag_datum(/datum/antagonist/nightmare))
+			message_say = "FOR THE DARKNESS!"
+		else if(our_guy.has_antag_datum(/datum/antagonist/santa))
+			message_say = "FOR CHRISTMAS!" //The sheer absurdity of this
+		else if(our_guy.has_antag_datum(/datum/antagonist/separatist))
+			message_say = "FOR THE MOTHERLAND!"
+		else if(our_guy.has_antag_datum(/datum/antagonist/survivalist) || our_guy.has_antag_datum(/datum/antagonist/heartbreaker))
+			message_say = "FOR ME!"
+		else if(our_guy.has_antag_datum(/datum/antagonist/wishgranter))
```
Comment (tralezab) [inline code/game/objects/items/grenades/plastic.dm:162]: this `if else` chain is now twenty one checks long

can we just make antagonistic declaration message a variable on /datum/antagonist?

Comment (ExcessiveUseOfCobblestone) [reply code/game/objects/items/grenades/plastic.dm:162]: Yes this is what I was thinking

Comment (unit0016) [reply code/game/objects/items/grenades/plastic.dm:162]: Probably a smart optimization.  Going to ~~hit byond with a hammer~~ attempt.

**Comment by unit0016** [comment]
I've decided I have the brain cells of a coconut wrapped in tinfoil that's been kicked around like a soccer ball, landed in the gulf of mexico, drifted all the way to ireland, got kicked back into the water, and promptly settled at some sandbar nobody goes to.

..In other words grug brain no know `/proc/` with `var/list/` and `cross atom communication` SCARY FIRE EEK
Someone else can do better, no reason to leave this open

**Comment by interestingusernam3** [comment]
Sad!

**Pull request closed without merging**